Zend/zend_vm_def.h

```diff
@@ -4344,7 +4349,8 @@ ZEND_VM_C_LABEL(fcall_end):
 		HANDLE_EXCEPTION();
 	}
 
-	ZEND_VM_SET_OPCODE(opline + 1);
+	CHECK_SYMBOL_TABLES()
+	OPLINE = opline + 1;
```

Comment:
Nit: This could be hidden in a `ZEND_VM_SET_OPCODE_NO_INTERRUPT(opline)` or `ZEND_VM_SET_OPCODE_EX(opline, interrupt)` macro